_projects/optimize_caching.md

```diff
@@ -62,23 +69,24 @@ increase performance even further.
 
 ##### CODING PHASE 1
 
-* The Proposed change to not explore unchanged directories has been  
-  implemented.
-* Thorough tests have been performed to concretely ascertain a measurable  
```

Comment:
I'm not keen on this being moved out of the project coding phase 1, especially not out of the project brief.

moving it now is planning to fail.

You should be trying to achieve this.

If you fall short, the mentors might push this task into coding phase 2.

Reply:
I have already discussed this with @Makman2 ignore directories functionality will be implemented during phase 2 also we are still not clear on a few implementation details so we had decided to take this up in phase2.

Also the same timeline was mentioned in my GSoC proposal and the cEP which got merged so I think we should keep the timelines consistent across all proposals

Reply:
Adding some further clarification as to why it was moved out of phase 1:
The original description included section based caching for phase2 since that was ruled during discussion before GSoC see: https://github.com/coala/coala/issues/2699#issuecomment-374154490 and the ignore directories depending upon factory class implementation it was moved to phase 2 and replaced by factory classes implementation in phase 1 which has been done and waiting to be approved See: https://github.com/coala/coala/pull/5485  & https://github.com/palash25/coala/pull/6

Reply:
This item was about measuring performance at beginning of project , so comparisons can be made after changes are made.
This is mandatory for verifying the project is successful in each phase

Reply:
I'm sorry I misunderstood. I thought you were talking about the first point being removed.